main.go

```diff
@@ -5,6 +5,7 @@ import (
 	models "github.com/moesif/moesifapi-go/models"
 	moesifmiddleware "github.com/moesif/moesifmiddleware-go"
 	options "github.com/moesif/moesifmiddleware-go-example/moesif_options"
+	"io"
```

Comment:
@bakennedy we need to import `"io/ioutil"` package which has `ReadAll` function we're using later to print the body. Otherwise this app won't start.